src/components/Header/Menu/index.js

```diff
@@ -0,0 +1,44 @@
+import { useState } from "react";
+import styled from "styled-components";
+//import { LanguageSwitcher } from "../../../components/Shared";
+
+const Menu = () => {
+  const menuItems = ["Home", "Orders", "Contact", "FAQ"];
+  const [active, setActive] = useState(true);
+
+  const handleActiveClass = (event) => {
+    setActive(event.target.innerText);
+  };
+  return (
+    <>
+      <MenuStyled onClick={handleActiveClass}>
+       
+        {menuItems.map((item, index) => (
+          <MenuItemStyled
+            className={`${active === item ? "activeMenu" : ""}`}
+            key={item + index}
```

Comment:
The key needed to identify which exact elements from similar ones need to be re-rendered, indexes, and other values that can be easily changed as keys its a bad practice.

https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/